src/PockyBot.NET/PockyBot.cs

```diff
@@ -31,7 +35,12 @@ public void Respond(Message message)
                 responder = _triggers.FirstOrDefault(x => _triggerResponseTester.ShouldTriggerInDirectMessage(message, x));
```

Comment:
Is there a default responder? Something that can decide what to do if none of the pre-coded responders match? For example, if there is a typo and they write `@PockyBot upadte`